woocommerce/woocommerce-android
# [Login] Improve site discovery to support sites with multiple Link headers

Closes: #12079 


### Description
This PR simply updates FluxC to include the changes of https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/3055, the FluxC PR improves the site discovery to support sites that send multiple Link headers.

### Testing information
1. To prepare the website, please install the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin, then add the following snippet:
```PHP
function modify_pre_get_shortlink_defaults($return, $id, $context, $allow_slugs) { 
	if ($id == 0) {
		return home_url( '/' );
	}
    return $return; 
}
// Add the filter to return the `shortlink` type as the `Link` header for the home page as well
add_filter( "pre_get_shortlink", "modify_pre_get_shortlink_defaults", 10, 4 );
```
2. Open the app and sign in to the website using site credentials.
3. Confirm the sign in works without issues.
